import-service/src/functions/importFileParser/handler.ts

```diff
@@ -0,0 +1,82 @@
+import {
+  formatJSONResponse,
+} from "@libs/api-gateway";
+import { middyfy } from "@libs/lambda";
+import { S3Event} from 'aws-lambda';
+import * as AWS from 'aws-sdk';
+import csv from 'csv-parser';
+
+const importFileParser = async (event: S3Event) => {
+  console.log('start process importProductsFile', JSON.stringify(event.Records));
+
+  const BucketName = process.env.BUCKET_NAME;
+  const s3 = new AWS.S3({ region: 'eu-west-1' });
```

Comment:
it is better to pass dependencies as parameters to lambda function.
see details here https://docs.aws.amazon.com/lambda/latest/dg/best-practices.html#function-code